### PR TITLE
feature(argus.backend.service.stats): Stats rework

### PR DESCRIPTION
**Pull request by k0machi**
This commit reworks stats gathering mechanism, while keeping the legacy
one for now, while also adding this mechanism for ReleaseStats and
TestDashboard, which is now separate from stats and is an independent
component.
[Trello](https://trello.com/c/FSyZhMeV/5011-show-not-planned-jobs)

### CODE REVIEW

**Comment by roydahan** [comment]
Putting this on-hold, till @k0machi will add few missing parts and be ready for deploy

**Review by bentsi — CHANGES_REQUESTED** [review]
need to remove old code (sidebar and release planner)

File: argus/backend/service/stats.py [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+from argus.backend.db import ScyllaCluster
+
+
+class ReleaseStats:
+    def __init__(self, release: ArgusRelease) -> None:
+        self.release = release
+        self.groups: list[GroupStats] = []
+        self.status_map = {
+            **{status: 0 for status in TestStatus}
+        }
```
Comment (bentsi) [inline argus/backend/service/stats.py:14]:
```suggestion
        self.status_map = {status: 0 for status in TestStatus}
```

Comment (k0machi) [reply argus/backend/service/stats.py]: Done

**Review by bentsi — APPROVED** [review]

**Pull request merged**